shared/redux/rtk-apis/assignments/assignments.api.ts

```diff
@@ -73,6 +78,20 @@ const assignmentsApi = projectApi.injectEndpoints({
       },
       invalidatesTags: ["Assignments"],
     }),
+
+    getAssignmentSubmissions: builder.query<StudentResponseDto[], number>({
+      query: (assignmentId) => ({
+        url: `/assignments/${assignmentId}/submissions`,
+        method: "GET",
+      }),
+      transformResponse: (response: IApiSubmissionResponse[]): StudentResponseDto[] =>
+        response.map((submission) => ({
+          firstName: submission.student.user.firstName.trim(),
+          lastName: submission.student.user.lastName.trim(),
+          fileUrl: submission.fileUrl,
+          createdAt: submission.createdAt,
+        })),
+    }),
```

Comment:
Don't we need cache invalidation here?

Reply:
Bhaiya, as I am fetching submissions, no need for invalidation now.